code/game/objects/items/GriffeningPorted.dm

```diff
@@ -0,0 +1,805 @@
+GLOBAL_LIST(CardCreatures)
+GLOBAL_LIST(CardEffects)
+GLOBAL_LIST(CardEquipment)
+GLOBAL_LIST(CardAreas)
+
+/obj/item/griffeningdeck
+	name = "deck of griffening cards"
+	desc = "A deck of griffening playing cards."
+	icon = 'icons/obj/toy.dmi'
+	var/deckstyle = "nanotrasen"
+	icon_state = "deck_nanotrasen_full"
+	w_class = WEIGHT_CLASS_SMALL
+	var/amountofcards //I hate this
+	var/IsBooster = FALSE //Muh booster packs
+	var/CardsGenerated = 0 //How many cards to make if it's a booster?
+	var/CardThatGenerated //Used for the procs down below
+	var/cooldown = 0
+	var/list/cards = list()
+
+/obj/item/griffeningdeck/examine(mob/user)
+	var/amountofcards = cards.len
+	to_chat(user, "<b>[src] has [amountofcards] cards.</b>")
+	..()
+
+/obj/item/griffeningdeck/proc/generate_creature()
+	CardThatGenerated = pick(GLOB.CardCreatures)
+	var/A = new CardThatGenerated
+	cards += A
```

Comment:
This could have just been cards += new CardThatGenerated, there was no need to reference the generated card at all